examples/shell/nrf52/.gn

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2020 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# The location of the build configuration file.
+buildconfig = "//../build/config/BUILDCONFIG.gn"
```

Comment:
Why the //.. ? I don't think this is useful because as soon as you import a common file //build is going to have to work.

Reply:
Unfortunately, I haven't managed to create cross-platfrom build script yet. So this is unused draft version of the file. I'll remove it from the repo now.

Reply:
> Unfortunately, I haven't managed to create cross-platfrom build script yet. So this is unused draft version of the file. I'll remove it from the repo now.

I would also suggest uploading the VSCode changes and any other unrelated changes separately.

Reply:
Ok. Should I remove them from this PR now?

Reply:
> Ok. Should I remove them from this PR now?

I think that's a good idea, yes.